### PR TITLE
Test client serving

### LINKED ISSUE #617: Web server fails when serving qxapp
image

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Test added to check we don't get errors when accessing the frontend app.

closes #617 

## Checklist

- [ ] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet

### CODE REVIEW

File: services/web/server/tests/unit/test_rest.py [hunk 1]
```diff
@@ -70,6 +70,16 @@ def client(loop, aiohttp_unused_port, aiohttp_client, api_specs_dir):
     assert data['name'] == 'simcore_service_webserver'
     assert data['status'] == 'SERVICE_RUNNING'
 
+async def test_check_serve_root(client):
+    resp = await client.get("/")
+    payload = await resp.json()
```
Comment (pcrespov) [inline services/web/server/tests/unit/test_rest.py]: This is `` payload = await resp.text() ``

File: services/web/server/tests/unit/test_rest.py [hunk 2]
```diff
@@ -70,6 +70,16 @@ def client(loop, aiohttp_unused_port, aiohttp_client, api_specs_dir):
...
+
+    assert resp.status == 200, str(payload)
+    data, error = tuple(payload.get(k) for k in ('data', 'error'))
+
+    assert data
```
Comment (pcrespov) [inline services/web/server/tests/unit/test_rest.py]: why don't you check the content of data as well?

File: ops/travis/system-testing/tests/test_swarm_runs.py [hunk 3]
```diff
@@ -165,3 +166,24 @@ def test_all_services_up(docker_client, services_docker_compose, tools_docker_co
         "Expected running, got \n{}\n{}".format(
                 pformat(task),
                 get_failed_tasks_logs(running_service, docker_client))
+
+
+async def test_check_serve_root():
+    req = urllib.request.Request("http://localhost:9081/")
+    try:
+        resp = urllib.request.urlopen(req)
+        print("Reason: ", resp)
```
Comment (sanderegg) [inline ops/travis/system-testing/tests/test_swarm_runs.py]: question: why don't you check both the response and payload in the one test?
and also is there no status code to check?
you can check doing:
```python
response = urllib.request.urlopen(req)
assert response.getcode() == 200
```

One other thing is that the swarm takes some time to be up (especially on travis), and we need to wait for the services. The test above actually waits. you may need to do the same before testing.

Comment (pcrespov) [reply ops/travis/system-testing/tests/test_swarm_runs.py]: Please do not add print in the tests ...

**Comment by pcrespov** [comment]
@odeimaiz I guess we forgot about this one? :-(

**Comment by odeimaiz** [comment]
wish granted @pcrespov

**Pull request merged**